Earthfile

```diff
@@ -146,6 +146,16 @@ earth-docker:
     COPY --build-arg VERSION=$TAG +earth/earth /usr/bin/earth
     SAVE IMAGE --push earthly/earth:$TAG
 
+# we abuse docker here to distribute our binaries
+prerelease-docker:
+    ARG EARTHLY_GIT_HASH
+    COPY +earth/earth /earth-linux-amd64
+    COPY --build-arg GOOS=darwin \
+        --build-arg GOARCH=amd64 \
+        --build-arg GO_EXTRA_LDFLAGS= \
+    +earth/earth /earth-darwin-amd64
+    SAVE IMAGE --push alexcb132/earthlybinaries:latest
```

Comment:
I'll make an earthly repo for this.

Reply:
I've created this: https://hub.docker.com/repository/docker/earthly/earthlybinaries

Reply:
thank you kindly :)